src/main/java/com/ft/methodearticlemapper/transformation/EomFileProcessor.java

```diff
@@ -60,9 +60,14 @@
         PREVIEW
     }
 
+    interface Type {
+//        String ARTICLE = "http://www.ft.com/ontology/content/Article";
```

Comment:
http://git.svc.ft.com/projects/CP/repos/content-public-read/browse/src/main/java/com/ft/contentpublicread/transform/TypeResolver.java

This is the "arcane" logic which differentiates an Article from Content at the moment. Luke suggested moving this to the mappers as there might be extra Methode flags available here in order to make the type determination less brittle (like in the case of ContentPackages where we have the isContentPackage flag)